api/envoy/config/route/v3/route_components.proto

```diff
@@ -224,6 +224,13 @@ message FilterAction {
   google.protobuf.Any action = 1;
 }
 
+// This can be used in route matcher :ref:`VirtualHost.matcher <envoy_v3_api_field_config.route.v3.VirtualHost.matcher>`.
+// When the matcher matches, routes will be matched and run.
+message RouteListAction {
```

Comment:
Sorry maybe you missed my comment before. WDYT about renaming RouteListAction -> RouteList so it more naturally matches "Route"? As long as we are adding a new thing I think that is better?

/wait

Reply:
Ah, I misunderstand the meaning of the second half of the comment. Yeah, this makes sense. I already updated it. Please take another look. Thank you!